### PR TITLE
[MRG +1] Allow int eventtype in eeglab epoch reader.

### PR DESCRIPTION
**Pull request by jaeilepp**
See https://github.com/mne-tools/mne-python/issues/2672#issuecomment-273058747

I've never really used the EEGLAB files so I'm not sure if the fix is the right one. I simply cast the int to a string. @jona-sassenhagen @jasmainak

### CODE REVIEW

**Comment by mmagnuski** [comment]
That was previously fixed for raw data in #3279 and #3354. Maybe epochs reader should also use `_read_eeglab_events` (it also contains other fixes like the case when there is only one event which raises `TypeError: 'mat_struct' object is not iterable`)?

**Comment by jaeilepp** [comment]
I have no experience on eeglab data, so I'm not able to tell which parts of ``_read_eeglab_events`` are reusable. I'll leave that to you guys.

File: mne/io/eeglab/eeglab.py [hunk 1]
```diff
@@ -484,6 +484,8 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             ev_idx = 0
             warn_multiple_events = False
             for ep in eeg.epoch:
+                if isinstance(ep.eventtype, int):
+                    ep.eventtype = str(ep.eventtype)
```
Comment (jasmainak) [inline mne/io/eeglab/eeglab.py]: @jaeilepp the `eventtype` field is like the `key` in our `event_id` dict. The only thing is that EEGLAB has it associated with every epoch

**Comment by jasmainak** [comment]
Fix looks reasonable to me. +1 for merge

**Comment by agramfort** [comment]
please update what's new and merge.

thx @jaeilepp for the quick fix

**Comment by agramfort** [comment]
I guess this is good enough

**Pull request merged**